pkgs/development/python-modules/pyside/tools.nix

```diff
@@ -13,7 +13,8 @@ buildPythonPackage rec {
 
   enableParallelBuilding = true;
 
-  buildInputs = [ cmake pyside qt4 pysideShiboken ];
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ pyside qt4 pysideShiboken ];
```

Comment:
`pyside` and `pysideShiboken` should be in `propagatedBuildInputs`.
(qt4 can stay though)

Reply:
Looks like that was already fixed in master by now.